components/table/demo/table-test.js

```diff
@@ -147,6 +147,19 @@ class TestTable extends RtlMixin(DemoPassthroughMixin(TableWrapper, 'd2l-table-w
 		this._sortDesc = !desc;
 	}
 
+	_renderDoubleSortButton(fruit1, fruit2) {
+		return html`
+			<th scope="col">
+				<d2l-table-col-sort-button nosort>
+					${fruit1}
+				</d2l-table-col-sort-button>
+				<d2l-table-col-sort-button nosort>
+					${fruit2}
+				</d2l-table-col-sort-button>
+			</th>
+		`;
+	}
```

Comment:
It would be useful if this were more functional in order to be able to test the impact of clicking the button.